### PR TITLE
Bug fix/retrieve past activities on page load

### PR DESCRIPTION
**Pull request by knees4bees**
## **_PR TEMPLATE_**
1. Is this a fix or a feature?
* Fix

2. What is the change?
* Rearrange code so that past activities display on DOM when page reloads (in order!)
* Clean up comments
* Move hard-coded styles out of `main.js` and into `styles.css`

3. What does it fix?
* The past activities used to not show up on the DOM even though they existed in the `pastActivities` array

4. Where should the reviewer start?
* In `Activity.js`, `styles.css`, and `main.js`, look at the diffs -- most changes are just comments being removed.
* In `main.js` and `Activity.js`, follow the logic logging a current activity and saving it to storage, then retrieving all the activities in storage and using them to populate the `pastActivities` array, then displaying those past activities to the DOM right away and also on page load.

5. How should this be tested?
* Create and log several activities (make sure the circle is the right color each time), then reload the page and log a few more activities. Check that the panel of past activities looks right.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -52,29 +52,51 @@ var pastActivities = [];
...
   startTimerButton.classList.add('cannot-click');
   currentActivity.startTimer();
 });
+
 logActivityButton.addEventListener('click', logActivityEvents);
+
 createNewActivityButton.addEventListener('click', returnToActivityForm);
 
+// Add functions below
 window.onload = function() {
```
Comment (stevecalla) [inline main.js:68]: Nice work creating the appropriate logic in the windows load function. That was tricky.

File: main.js [hunk 2]
```diff
@@ -52,29 +52,51 @@ var pastActivities = [];
...
 function populatePastActivities() {
+  pastActivities = [];
   var keys = Object.keys(localStorage);
-  for (var i = 0; i < keys.length; i++) {
-    pastActivities.push(localStorage.getItem(keys[i]));
+  sortDescending(keys);
+  reviveActivitiesFromStorage(keys);
+}
+
+function sortDescending(array) {
```
Comment (stevecalla) [inline main.js:86]: Nice addition sorting in ascending order.

File: styles.css [hunk 3]
```diff
@@ -110,6 +110,18 @@ timer-box-wrapper {
   width: 17vw;
 }
 
+.circle-outline-study {
```
Comment (stevecalla) [inline styles.css:113]: Good refactor.

File: main.js (hunk 1 above)
Comment (knees4bees) [reply main.js:68]: Thanks! I just realized we still have a little cleanup to do here, which we can address when we do the cleaning pass tomorrow. We'll need to get rid of the `console.log` statements and the alert in this `onload` function.

**Pull request merged**